### ISSUE TITLE
Throwing exception with Craft 4.12.0

### ISSUE THREAD

**Issue by jripmeester** [nystudio107/craft-templatecomments#45]
### Describe the bug
There looks to be a bad interaction with the Craft 4.12.0 update

### To reproduce
Steps to reproduce the behaviour:
1. Upgrade to Craft 4.12.0
2. Visit front-end page
3. error shows
```
TypeError: count(): Argument #1 ($value) must be of type Countable|array, null given in /var/www/html/vendor/twig/twig/src/Parser.php:312
Stack trace:
#0 /var/www/html/vendor/twig/twig/src/Extension/CoreExtension.php(1924): Twig\Parser->hasInheritance()
#1 /var/www/html/vendor/twig/twig/src/ExpressionParser.php(476): Twig\Extension\CoreExtension::parseParentFunction(Object(nystudio107\templatecomments\web\twig\TemplateCommentsParser), Object(Twig\Node\Node), Object(Twig\Node\Node), 5)
#2 /var/www/html/vendor/twig/twig/src/ExpressionParser.php(240): Twig\ExpressionParser->getFunctionNode('parent', 5)
#3 /var/www/html/vendor/twig/twig/src/ExpressionParser.php(177): Twig\ExpressionParser->parsePrimaryExpression()
#4 /var/www/html/vendor/twig/twig/src/ExpressionParser.php(72): Twig\ExpressionParser->getPrimary()
#5 /var/www/html/vendor/nystudio107/craft-templatecomments/src/web/twig/TemplateCommentsParser.php(194): Twig\ExpressionParser->parseExpression()
#6 /var/www/html/vendor/nystudio107/craft-templatecomments/src/web/twig/tokenparsers/CommentBlockTokenParser.php(60): nystudio107\templatecomments\web\twig\TemplateCommentsParser->subparse(Object(Closure), true)
#7 /var/www/html/vendor/nystudio107/craft-templatecomments/src/web/twig/TemplateCommentsParser.php(237): nystudio107\templatecomments\web\twig\tokenparsers\CommentBlockTokenParser->parse(Object(Twig\Token))
#8 /var/www/html/vendor/nystudio107/craft-templatecomments/src/web/twig/tokenparsers/CommentsTokenParser.php(46): nystudio107\templatecomments\web\twig\TemplateCommentsParser->subparse(Array, true)
#9 /var/www/html/vendor/nystudio107/craft-templatecomments/src/web/twig/TemplateCommentsParser.php(237): nystudio107\templatecomments\web\twig\tokenparsers\CommentsTokenParser->parse(Object(Twig\Token))
#10 /var/www/html/vendor/nystudio107/craft-templatecomments/src/web/twig/TemplateCommentsParser.php(149): nystudio107\templatecomments\web\twig\TemplateCommentsParser->subparse(NULL, false)
#11 /var/www/html/vendor/twig/twig/src/Environment.php(523): nystudio107\templatecomments\web\twig\TemplateCommentsParser->parse(Object(Twig\TokenStream))
#12 /var/www/html/vendor/twig/twig/src/Environment.php(551): Twig\Environment->parse(Object(Twig\TokenStream))
#13 /var/www/html/vendor/craftcms/cms/src/web/twig/Environment.php(39): Twig\Environment->compileSource(Object(Twig\Source))
#14 /var/www/html/vendor/twig/twig/src/Environment.php(381): craft\web\twig\Environment->compileSource(Object(Twig\Source))
#15 /var/www/html/vendor/twig/twig/src/Environment.php(343): Twig\Environment->loadTemplate('__TwigTemplate_...', 'index')
#16 /var/www/html/vendor/twig/twig/src/Environment.php(306): Twig\Environment->load('index')
#17 /var/www/html/vendor/craftcms/cms/src/web/View.php(495): Twig\Environment->render('index', Array)
#18 /var/www/html/vendor/craftcms/cms/src/web/View.php(548): craft\web\View->renderTemplate('index', Array)
#19 /var/www/html/vendor/craftcms/cms/src/web/TemplateResponseFormatter.php(57): craft\web\View->renderPageTemplate('index', Array, 'site')
#20 /var/www/html/vendor/yiisoft/yii2/web/Response.php(1109): craft\web\TemplateResponseFormatter->format(Object(craft\web\Response))
#21 /var/www/html/vendor/craftcms/cms/src/web/Response.php(339): yii\web\Response->prepare()
#22 /var/www/html/vendor/yiisoft/yii2/web/Response.php(340): craft\web\Response->prepare()
#23 /var/www/html/vendor/yiisoft/yii2/base/Application.php(390): yii\web\Response->send()
#24 /var/www/html/public_html/index.php(12): yii\base\Application->run()
#25 {main}
```

Error seems to be the same as mentioned here:  #44 

### Versions
- Plugin version: 4.0.2
- Craft version: 4.12.0

**Comment by khalwat** [comment]
Ugh, okay, they apparently updated to Twig `3.12.0` for Craft 4.x as well. I'll get it.

**Comment by khalwat** [comment]
Released as `4.0.3` https://github.com/nystudio107/craft-templatecomments/releases/tag/4.0.3
